MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly

```diff
@@ -0,0 +1,189 @@
+\version "2.18.2"
+
+\header {
+  title = "Nativity of the Theotokos"
+  subtitle = "Antiphon"
+  subsubtitle = "First"
+  copyright = "This arrangement is free to distribute for Liturgical use"
+  tagline = "Arranged using https://oca.org/liturgics/learning-the-tones"
+}
+
+#(set-default-paper-size "letter")
+
+% Provide an easy way to group a bunch of text together on a breve
+% http://lilypond.org/doc/v2.18/Documentation/notation/working-with-ancient-music_002d_002dscenarios-and-solutions
+recite = \once \override LyricText.self-alignment-X = #-1
+
+\defineBarLine "invisible" #'("" "" "")
+global = {
+  \time 1/1 % Not used, Time_signature_engraver is removed from layout
+  \key g \major
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+refrain = \lyricmode {
+  Through the in -- ter -- ces -- sions of the The -- o -- to -- kos, __ O __ Sav -- ior, save __ us.
+}
+verseOne = \lyricmode {
+  % The verses can be broken up further than they are in the Liturgy variable text
+  % http://ww1.antiochian.org/sites/default/files/nativity_of_theotokos_series.pdf
+  Re -- mem -- ber, O Lord, Da -- vid and all his meek -- ness.
+  \refrain
+  Lo, we have heard it at Eph -- ra -- tha.
+    We have found it in the plains of the wood.
+  \refrain
+  Glo -- ri -- ous things are spo -- ken of thee,
+    O ci -- ty of God.
+  \refrain
+  God is in the midst of her; she shall not be shak -- en;
+    God shall help her right ear -- ly in the morn -- ing.
+  \refrain
+  Glo -- ry to the \recite "Father and to the Son and to" the Ho -- ly Spir -- it,
+    both now and \recite "ever, and unto ages" of ag -- es. A -- men.
+  \refrain
+}
+
+soprefmost = { g4 g g a b4 b4 b4 b4 a b c2 b4( c) b( a) g2 a2 b1( a) g1 }
+sopref = { \soprefmost \bar "||" }
+soprano = \relative g' {
+  \global
+  % Verse 1
+  g4 g g a b c2. b4 r4 \bar "|"
+  a4 b g a2. g2 \bar "||"
+  \sopref % Refrain
+  % Verse 2
+  g4 g a b a b c2. c4 b2 r4 \bar "|"
+  g4 a b b b g a2 a4 a g2. \bar "||"
+  \sopref % Refrain
+  % Verse 3
+  g4 g a b b c2. c4 c b2 r4 \bar "|"
+  g4 a2 a4 a g2. \bar "||"
+  \sopref \allowPageTurn % Refrain
+  % Verse 4
+  g4 g g a b b b b b a b c2. b4 r4 \bar "|"
+  g4 a b b b b b b g a2. g4 \bar "||" \allowPageTurn
+  \sopref % Refrain
+  \allowPageTurn % GNE
+  g4 g g a b\breve b4 a4 b4 c2. b4 r4 \bar "|"
+  g4 g a b\breve g4 a2 a4 a g2 r4 \bar "||"
+  \soprefmost \bar "|." % Refrain
+}
+
+altrefmost = { g4 g g g g g g g g g g2 g2 g g g2 g1( d) d1 }
+altref = { \altrefmost \bar "||" }
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g2. g4 r4 \bar "|"
+  g4 g g fis2. d2 \bar "||"
+  \altref % Refrain
+  % Verse 2
+  g4 g g g g g g2. g4 g2 r4 \bar "|"
+  g4 g g g g g g2 g4 g d2. \bar "||"
```

Comment:
Still need to change last 3 Gs to F#s, like this:
g4 g g g g g fis2 fis4 fis d2. \bar "||"